internal/pkg/source/entry_test.go

```diff
@@ -310,3 +335,183 @@ func getFieldKindToValue(cfg *config.Config, entries []string) map[config.FieldK
 
 	return fieldKindToValue
 }
+
+type TimeFormattingTestCase struct {
+	TestName       string
+	JsonString     string
+	ExpectedOutput string
+}
+
+func getTimestampFormattingConfig(fieldKind config.FieldKind) *config.Config {
+	return &config.Config{
+		Path: config.PathDefault,
+		Fields: []config.Field{{
+			Title:      "Time",
+			Kind:       fieldKind,
+			References: []string{"$.timestamp", "$.time", "$.t", "$.ts"},
+			Width:      30,
+		}},
+	}
+}
+func TestSecondTimeFormatting(t *testing.T) {
+	t.Parallel()
+
+	cfg := getTimestampFormattingConfig(config.FieldKindSecondTime)
+
+	expectedOutput := time.Unix(1, 0).Format(time.RFC3339)
+
+	secondsTestCases := []TimeFormattingTestCase{{
```

Comment:
```suggestion
	secondsTestCases := [...]TimeFormattingTestCase{{
```

just cosmetic, it defines an array, not slice